Core/SiteRating.swift

```diff
@@ -26,7 +26,7 @@ public class SiteRating {
     }
 
     public var domain: String? {
-        return url.host
+        return url.host?.punycodeDecodedHostname
```

Comment:
This var is used mainly for display to the user so should show the punycode decoded version.